test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -217,3 +217,33 @@ func TestMeshGatewaySecure(t *testing.T) {
 		})
 	}
 }
+
+const consulMemberStatusAlive = 1
```

Comment:
... why 😂

Reply:
It's for @ashwin-venkatesh 😄 

But seriously, there are 4 integers values for member status, and it wasn't super clear to me which one is which (is 0 good or bad?). So this is more for clarity also around consul api, which doesn't have them as an enum.

Reply:
I should prob move it inside the function though. It doesn't need to live here.

Reply:
The shade I receive because I ask for a few constants every now and then 😛

Reply:
I found out that these constants exist in serf, so I switched it to use `serf.StatusAlive`. That way we get the best of both worlds.

Reply:
No no! :D I like the constant. I was saying why is this an int in consul's API instead of a bool. But now I hear it's an enum so I guess that makes sense, although really a string is better IMO.